Gemfile

```diff
@@ -24,8 +24,9 @@ gem "bootsnap", ">= 1.4.2", require: false
 gem "aws-sdk-sqs"
 gem "doorkeeper", "~> 5.6"
 gem "dry-validation"
-gem "faraday", "~> 2.7.4"
-gem "faraday-retry", "~> 2.2.0"
+gem "faraday", "~> 2.12"
+gem "faraday-net_http_persistent", "~> 2"
+gem "faraday-retry", "~> 2.2"
 gem "hmcts_common_platform", github: "ministryofjustice/hmcts_common_platform", tag: "v0.2.0"
```

Comment:
These are just Gem updates, the new Faraday brings in some improvement and bug fix, which are good to have!